packages/terra-button-group/src/ButtonGroup.jsx

```diff
@@ -27,6 +27,10 @@ const propTypes = {
    * A list of keys of the buttons that should be selected.
    */
   selectedKeys: PropTypes.arrayOf(PropTypes.string),
+  /**
+   * Callback function triggered when button gains focus.
+   */
+  onFocus: PropTypes.func,
```

Comment:
Should this prop be added? I'm not seeing where it is being used

Reply:
It was added to avoid multiple onFocus events, for when the focus handler was added to both the ButtonGroup and ButtonGroupButton.

Reply:
I'm not sure I understand 100%. Did you add this to remove the onFocus from being used if a consumer passes it in? or will this be triggered instead?

Reply:
I added this thinking that if ButtonGroupButton and ButtonGroup both have onFocus prop, then ButtonGroup should not trigger an onFocus event because both will be calling it on the same target. But I didn't realize that it would also be inhibiting consumers from doing any onFocus events on the ButtonGroup.

Removed it here https://github.com/cerner/terra-core/pull/2933/commits/b5c32f607c81526c5b75e4fa4052a194427aa5c2